src/tools/web/README.md

```diff
@@ -114,6 +114,53 @@ Elektra web consists of multiple components:
 - [elektrad](https://master.libelektra.org/doc/api_blueprints/elektrad.apib), documentation: https://elektrad.docs.apiary.io/
 - [webd](https://master.libelektra.org/doc/api_blueprints/webd.apib), documentation: https://elektrawebd.docs.apiary.io/
 
+## Test REST API on localhost
+
+In order to test API on localhost, your have to start elektrad instance. You can do it in two ways:
+
+- run manually
+
+  - `cd libelektra/src/tools/web`
+  - `cd elektrad`
+  - `npm install`
+  - `npm start` (replaces `kdb run-elektrad`)
+
+- by installing elektrad tool together with Elektra and run it
+  - please see the documentation [here](https://www.libelektra.org/tools/web) on how to install and run the elektra-web tool
```

Comment:
Is this self-reference on purpose?

Reply:
oh sorry, I didn't notice that, I will improve this